piccolo/columns/defaults/uuid.py

```diff
@@ -27,6 +27,10 @@ def cockroach(self):
     def sqlite(self):
         return "''"
 
+    @property
+    def mysql(self):
+        return f"'{uuid.uuid4()}'"
```

Comment:
I see where you're coming from. but ideally we need a different solution here. This will give each row the same value by default.

MySQL has a `uuid()` function, but it's uuid v1.

```
SELECT uuid();
```

https://dev.mysql.com/doc/refman/8.4/en/miscellaneous-functions.html#function_uuid

We could define our own uuid4 function in the database, or just default to an empty string.